R/busy-indicators.R

```diff
@@ -0,0 +1,229 @@
+#' Enable/disable busy indication
+#'
+#' To enable/disable busy indicators, include the result of this function in the
+#' app's UI.
```

Comment:
Consider rewriting this to be more user-facing and to fully describe the feature.